tests/qemu-tests/tests/system/test_commands_kernel.py

```diff
@@ -86,3 +86,25 @@ def get_slab_object_address():
         if len(matches) > 0:
             return (matches[0], cache_name)
     raise ValueError("Could not find any slab objects")
+
+
+def test_command_msr_read():
+    if pwndbg.aglib.arch.name not in ["x86", "x86-64"]:
+        return
```

Comment:
See https://docs.pytest.org/en/stable/reference/reference.html#pytest-skip
```suggestion
       pytest.skip("Unsupported architecture: msr tests only work on x86 and x86-64")
```

Reply:
This may require importing pytest if its not imported before